pkgs/applications/networking/cluster/k8sgpt/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "k8sgpt";
+  version = "0.2.9";
+
+  src = fetchFromGitHub {
+    owner = "k8sgpt-ai";
+    repo = "k8sgpt";
+    rev = "v${version}";
+    hash = "sha256:039yiwyh84l1dm6b4vzabv427kfpxj41bjs53cqq85xmqah34bdm";
+  };
+
+  vendorHash = "sha256-nutpoc8UAPXgvBr4hUTYq9/d5OQXGOl3QNtaWhH9Cpw=";
```

Comment:
```suggestion
  vendorSha256 = "sha256-tXLt18ryxO9Q7E/MJWDlZy1gmxnMauqfcbmRdF9uUt8=";
```

Reply:
@ivankovnatsky Thanks for updating the hash, but I `vendorHash` is preferred. Please check this [comment](https://github.com/NixOS/nixpkgs/pull/222164#discussion_r1142076424)